numba_cuda/numba/cuda/target.py

```diff
@@ -61,6 +62,54 @@ def resolve_value_type(self, val):
 # Implementation
 
 
+class CUDACABIArgPacker(ArgPacker):
+    """
+    Subclass of ArgPacker which packs arguments according to the
+    CUDA C ABI calling convention.
+    """
+
+    def __init__(self, dmm, fe_args):
```

Comment:
Dumb question from someone who has little knowledge in Numba internal: As you said the subclass here looks (almost?) identical to `ArgPacker`. So is the C ABI information fully encoded in the `dmm` argument?